README.md

```diff
@@ -47,11 +50,11 @@ A series of Jenkins "build step" implementations are provided, which you can sel
 
 8. "Create OpenShift Resource(s)":  performs the equivalent of an `oc create` command invocation; this build step takes in the provided JSON or YAML text, and if it conforms to OpenShift schema, creates whichever OpenShift resources are specified.
 
-	- NOTE: if a namespace is specified for any of the resources, the plugin will attempt to create the resource in that namespace instead of the namespace specified in the "The name of the project to create the resources in" field.  However, the authorization token provided must have edit permission to that project/namespace.
+	- NOTE: if a namespace is specified in the provided json/yaml for any of the resources, that namespace will take precedence over the namespace specified in the "The name of the project to create the resources in" field.  However, the authorization token provided in the field "The authorization token for interacting with OpenShift" must have edit permission to any project/namespace reference in the json/yaml.  Or if no token is specified, the assumption is that Jenkins in running in OpenShift, and the default service account associated with the project/namespace Jenkins is running in must have edit permission.
```

Comment:
"permission to any project/namespace reference in the json/yaml."

referenced.